.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
does this run on pushes to any branch? we typically put in the `branches: main` for `push` event since i think that makes sure we only build for pushes to that branch instead of on pushes for any branch. not sure if that's how things actually behave, but i think it is.

Reply:
@samandmoore i just copied from delayed, but i guess that's something we should add?